monero-outreach-docs/translations/es/storyboard/wallet_quickstart_es.md

```diff
@@ -0,0 +1,56 @@
+
+Los monederos ligeros vienen en dos tipos: protección de claves-de-visualización y compartir claves-de-visualización. Los monederos ligeros de protección de claves de visualización mantienen tus claves de visualización completamente privadas, y por consecuencia tienen que descargar la cadena de bloques, o al menos la porción de ella después de la fecha de creación del monedero, para calcular el balance de Monero en el monedero. Los monederos ligeros de compartir claves de visualiación, por otro lado, comparten la clave de visualización con el servidor ejecutando el nodo remoto para calcular el balance de entrada del monedero en el servidor. Estos eficientes monederos son a veces llamados aplicaciónes de monedero. El compartir las claves de visualización reduce la privacidad si el servidor ejecutando el nodo de reconocimiento de clave de visualización no es confiable debido a que permite la obserbación de transacciones de entrada. Nótese, sin embargo, que algunas veces los dueños de monederos comparten sus claves de visualización para auditoría de todas formas — no da lugar para transferir Monero.
+
+Para máxima seguridad — seguridad incluso si tu dispositivo primario está comprometido por malware — se pueden utilizar monederos hardware especializados. Los monederos hardware son pequeños dispositivos electrónicos que guardan tus claves privadas y se comunican con software en el dispositivo primario usando sólo información pública. Los monederos hardware otorgan seguridad extra por el típico precio de $50-$150.
+
+Así que, con todo esto en mente, considera las siguientes opciones: si deseas ejecutar tu propio nodo completo, una buena elección es el monedero Monero GUI o CLI oficial. El primero provee una interfaz gráfica de usuario, y el último una interfaz textual. Ambos pueden ser configurados para ya sea ejecutar un nodo completo o conectarse a un nodo remoto en un modo mejorado de privacidad. Estos pueden ser descargados de [getmonero.org/es/downloads](https://getmonero.org/es/downloads/). Para dispositivos Android, Monerujo, en [monerujo.io](https://www.monerujo.io/), o para iOS, Cake Wallet, en [cakewallet.io](https://cakewallet.io/), son monederos ligeros de protección de claves de visualización. Si quisieras un monedero ligero de compartir claves de visualización, el monedero basado en navegador MyMonero, en [mymonero.com](https://mymonero.com/), se recomienda. The Edge Wallet, en [edge.app](https://edge.app/), es otra sólida opción de monedero ligero de compartir claves de visualización para ambos Android y iOS.
```

Comment:
Suggestion. to keep with the number wording:

- "El primero provee una interfaz gráfica de usuario, y el _**segundo**_ una interfaz textual. Ambos pueden ser configurados _**ya sea para**_ ejecutar un nodo completo o conectarse a un nodo remoto en un modo mejorado de privacidad."

Reply:
Thanks for noting that.